cogs/status.py

```diff
@@ -1,13 +1,15 @@
 from asyncio import wait_for
+
+import discord
 from discord import slash_command, option, Embed, utils as dutils
-from discord.ext import commands
 from discord.ext.commands import Bot
+
 from utilities.data import remove_colors_from_string, Colors, get_server_status
 from utilities.database import selector
 from utilities.utility import check_ip
```

Comment:
Here as well.

Reply:
Done to seperate third party imports (discord) from local imports (utilities.data) and built-in imports (asyncio)